src/util.rs

```diff
@@ -6,13 +6,18 @@ use std::any::type_name;
     clippy::unwrap_used,
     reason = "`last()` cannot return `None` since `type_name` always returns `&str`."
 )]
-pub fn get_type_name<T>() -> String {
```

Comment:
Ask about why the change

Reply:
This update was my take on @eywalker's [suggestion](https://github.com/walkerlab/orcapod/pull/44#discussion_r1951707177).